src/utilities/CheckUtility.ts

```diff
@@ -4,51 +4,34 @@ import StringUtility from './StringUtility';
 import IReplacerSlotQuestion from '../models/IReplacerSlotQuestion';
 import colors from 'colors';
 
-/**
- * https://timber.io/blog/creating-a-real-world-cli-app-with-node/#errors-and-exit-codes
- * @param message
- * @param isError
- */
-export const errorMessageAndExit = (isError: boolean, message: string) => {
-    if (isError) {
-        console.error(colors.bold.red(`[Error in generate-template-files]:`), colors.red(message));
-        process.exit(1);
-    }
-};
-
-export const isBooleanType = (value: any) => {
+export const isBooleanType = (value: unknown): value is boolean => {
     return typeof value === 'boolean';
 };
 
-/**
- * Helper function for throwing errors if a given expression evaluates to false.
- * This function is strict and will throw an error the the type of the first
- * argument is not "boolean".
- */
-export const errorIfFalse = (isError: boolean, errorMessage: string): Error | void => {
-    if (!isBooleanType(isError)) {
-        return errorIfFalse(false, `errorIfTrue() first argument must be a boolean but argument was of type ${typeof isError}`);
-    }
-
-    if (!isError) {
-        return new Error(errorMessage);
+export const displayError = (isError: boolean, errorMessage: string): Error | void => {
+    if (isError) {
+        try {
+            throw new Error(errorMessage);
+        } catch (error) {
+            console.info(colors.bold.red(`[Error in generate-template-files]: ${colors.red(error.message)}`));
+        }
```

Comment:
@knona @ccheney I didn't want to put a `try/catch` everywhere so I did this. Seems like hacky but it works.

Reply:
If you throw the error in the try block, you will always jump to the catch block. Therefore, the code is equivalent to the code in the catch block, without throwing an error. 
You should use the try...catch at the top of the call stack.
Otherwise the code won't break and will continue to run normally.
Ex: do a ctrl + c while an equirer prompt. 

You should set the try catch in the main methods which are "generate" and "commandLine". 
To me it's the best way to handle the errors even if it is a bit ugly.

Reply:
done